gateway/services/notifications.go

```diff
@@ -75,6 +75,12 @@ var NotificationsRoutes = arbor.RouteCollection{
 		"/notifications/{id}/info/",
 		alice.New(middleware.IdentificationMiddleware, middleware.AuthMiddleware([]models.Role{models.AdminRole})).ThenFunc(GetTopicInfo).ServeHTTP,
 	},
+	arbor.Route{
```

Comment:
This route will never be matched since `/notifications/{id}/` will match it first. So this need to be moved above that.